demos/card.html

```diff
@@ -0,0 +1,172 @@
+          <div class="mdl-card__supporting-text">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.
+          </div>
+        </div>
+
+        <div class="mdl-card demo-card demo-card--with-avatar">
+          <div class="mdl-card__primary">
+            <div class="demo-card__avatar"></div>
+            <div class="mdl-card--title">Title</div>
+            <div class="mdl-card--subtitle">Subhead</div>
+          </div>
+          <div class="mdl-card__media demo-card__16-9-media"></div>
+          <div class="mdl-card__supporting-text">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.
+          </div>
+          <div class="mdl-card__actions">
+            <button class="mdl-button">Action 1</button>
+            <button class="mdl-button">Action 2</button>
+          </div>
+        </div>
+
+        <div class="mdl-card demo-card demo-card--with-avatar">
+          <div class="mdl-card__primary">
+            <div class="demo-card__avatar"></div>
+            <div class="demo-card__text">
+              <div class="mdl-card--title">Title</div>
+              <div class="mdl-card--subtitle">Subhead</div>
+            </div>
+          </div>
+          <div class="mdl-card__media demo-card__16-9-media"></div>
+          <div class="mdl-card__actions mdl-card__actions--vertical">
+            <button class="mdl-button">Action 1</button>
+            <button class="mdl-button">Action 2</button>
+          </div>
+        </div>
+
+        <div class="mdl-card demo-card">
+          <div class="mdl-card__media demo-card__16-9-media"></div>
+          <div class="mdl-card__primary">
+            <div class="mdl-card--large-title">Title goes here</div>
+            <div class="mdl-card--subtitle">Subtitle here</div>
+          </div>
+          <div class="mdl-card__actions">
+            <button class="mdl-button">Action 1</button>
+            <button class="mdl-button">Action 2</button>
+          </div>
+        </div>
+
+        <div class="mdl-card demo-card">
+          <div class="mdl-card__primary">
+            <div class="mdl-card--large-title">Title goes here</div>
+            <div class="mdl-card--subtitle">Subtitle here</div>
+          </div>
+          <div class="mdl-card__supporting-text">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
+            dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
+            ea commodo consequat.
+          </div>
+          <div class="mdl-card__actions">
+            <button class="mdl-button">Action 1</button>
+            <button class="mdl-button">Action 2</button>
+          </div>
+        </div>
+
+        <style>
+          .demo-card--bg-demo {
+            height: 21.875rem; /* 350sp */
+            background-image: url("images/bamboo.jpg");
```

Comment:
These photos come from [unsplash](https://unsplash.com/license). That license should cover our use-case, right?

I'm happy to replace them with placekitten, but it may make things tricky if we ever decide to use the demos for visual diff testing.